wp-odm_related.php

```diff
@@ -142,9 +142,12 @@ public function related_save_post($post_ID)
           delete_post_meta($post_ID, 'related_content');
 
           $all_related_types = get_related_types();
-          foreach($all_related_types as $type):
-            delete_post_meta($post_ID, $type);
-          endforeach;
+
+					//if(isset($all_related_types) && !empty($all_related_types)):
```

Comment:
@Huyeng if this is commented, it does not introduce any change at all? Are you still working on this PR?